src/Symfony/Component/Translation/Translator.php

```diff
@@ -171,7 +171,11 @@ public function setLocale($locale)
      */
     public function getLocale()
     {
-        return $this->locale;
+        if (null === $this->locale) {
+            @trigger_error(sprintf('Using a "null" $locale is deprecated since Symfony 4.4. You should set it via %s::setLocale($locale) explicitly.', static::class, \E_USER_DEPRECATED));
```

Comment:
not sure this is needed, you should have gotten a deprecaiton in the first place for setting null.

Reply:
also we don't add new deprecations in bugfix-only branches

Reply:
I removed the deprecation.